sky/backends/cloud_vm_ray_backend.py

```diff
@@ -2930,19 +2922,16 @@ def _provision(
                         error_message = str(e)
 
                     if retry_until_up:
-                        logger.error(error_message)
-                        # Sleep and retry.
-                        gap_seconds = backoff.current_backoff()
-                        plural = 's' if attempt_cnt > 1 else ''
+                        gap_seconds = _RETRY_UNTIL_UP_INIT_GAP_SECONDS
```

Comment:
We lose the exponential backoff from before. Is there any easy way to preserve it?

Reply:
Actually there is no exponential backoff before because the `max_backoff_factor` is set to 1 https://github.com/skypilot-org/skypilot/blob/69977b14667db086d765bd8f6271080b263bc3a6/sky/backends/cloud_vm_ray_backend.py#L2894

It is possible to apply some potential backoff policy by passing the previous backoff interval in a contextual variable, but looks like we don't need such policy right now.